bberg/src/flavor_builder.rs

```diff
@@ -365,16 +368,30 @@ fn create_polynomial_views(first_poly: &String) -> String {
           using Base::Base;
       }};
   
-    class AllPolynomials : public AllEntities<Polynomial> {{
+    /**
+     * @brief A container for the prover polynomials handles.
+    */
+    class ProverPolynomials : public AllEntities<Polynomial> {{
       public:
-        [[nodiscard]] size_t get_polynomial_size() const {{ return this->{first_poly}.size(); }}
-        [[nodiscard]] AllValues get_row(const size_t row_idx) const
+        // Define all operations as default, except move construction/assignment
```

Comment:
@Maddiaa0 It does not look like that this comment is implemented:

`ProverPolynomials& operator=(ProverPolynomials&& o) noexcept = default;`

Reply:
@ludamad